### ISSUE TITLE
Inconsistent static path generation in brunch vs static_path/2

### ISSUE THREAD

**Issue by seantanly** [phoenixframework/phoenix#1499]
Running `mix phoenix.digest` uses brunch to generate and fingerprint the static assets into `priv/static`. 

However I noticed the brunch generated URLs in e.g. `app.css` 
```
background-image: url("/images/phoenix-5bd99a0d17dd41bc9d9bf6840abcc089.png");
```
does not contain `?vsn=d` which is required by Plug.Static to properly send far expiry `cache-control` headers. In this case, browser access will always trigger a request to the server, even if just requesting for a 304. 

This issue also results in duplicate downloads of the same asset if that asset is accessed in the regular HTML via `static_path/2` helper. E.g. `static_path(@conn, "/images/phoenix.png")`. The reason is due to one path containing `?vsn=d` and the other does not, resulting in the browser handing them as two different assets.

I'm not familiar with brunch, but the easiest way seems to be modifying brunch behavior to add `?vsn=d` to the fingerprinted URLs. If not, Plug.Static and `static_path/2` need to be modified to reconcile the difference with brunch generated URLs.

**Comment by josevalim** [comment]
I have pushed a fix to the v1.1 branch, can you please give it a try and let us know how it goes? Add this to your mix.exs:

    {:phoenix, github: "phoenixframework/phoenix", branch: "v1.1", override: true}

**Comment by seantanly** [comment]
That's fast! The fix works perfectly. :)
